### ISSUE TITLE
Vale Package renaming INI file in v3

### ISSUE THREAD

**Issue by napcs** [jdkato/txtlint#991]
### Check for existing issues

- [x] Completed

### Environment

@jdkato Just ran into this again on the latest Vale. It looked familiar and I did report this a couple years ago. #583 

The logic is the same as it was in v2... the rename is happening before the file move.

https://github.com/errata-ai/vale/blob/v3/cmd/vale/sync.go#L133-L138

Need to move the file first and then change its name.

### Describe the bug / provide steps to reproduce it

Create a local package with a styles directory and a .vale.ini file
Use a local path in vale.ini

```
Packages =  /Users/brianhogan/dev/vale-custom
```

The .vale.ini file gets renamed when you run `vale sync`.

**Comment by ex-daniel-graupner** [comment]
I can confirm, the same happens with relative package paths.
